### ISSUE TITLE
Warn about (or outright disable) apt-get upgrade

### ISSUE THREAD

**Issue by nf** [monome/norns#498]
If you run `apt-get update && apt-get upgrade` on your norns it will hose the custom kernel and related things that are required to drive the norns hardware (including the usb ports, and therefore wifi). To get out of this state requires you to [rewrite the filesystem](https://github.com/monome/norns-image/blob/master/readme-usbdisk.md) which is an annoyance.

It would be good if we could warn about, or even entirely prevent, this particular foot-gun. :-)

**Comment by tehn** [comment]
will investigate pinning the version number of the kernel so `apt-get` won't mess with it

**Comment by simonvanderveldt** [comment]
I've made a PR to norns-image to add this https://github.com/monome/norns-image/pull/57
We should probably include the same thing in the next update.
Note I see this as a temporary measure, we should make our own proper kernel package which replaces the upstream kernel package, I've created an issue for that https://github.com/monome/linux/issues/10

And finally: @tehn maybe we could/should merge the norns-image stuff into this repo? Not sure it makes sense to maintain multiple repos for this?
